unity-environment/Assets/ML-Agents/Scripts/Brain.cs

```diff
@@ -86,6 +89,15 @@ public class BrainParameters
  */
 public class Brain : MonoBehaviour
 {
+    // Current agent info
+    public Dictionary<int, List<float>> currentStates = new Dictionary<int, List<float>>();
+    public Dictionary<int, List<Camera>> currentCameras = new Dictionary<int, List<Camera>>();
+    public Dictionary<int, float> currentRewards = new Dictionary<int, float>();
+    public Dictionary<int, bool> currentDones = new Dictionary<int, bool>();
+    public Dictionary<int, float[]> currentActions = new Dictionary<int, float[]>();
+    public Dictionary<int, float[]> currentMemories = new Dictionary<int, float[]>();
+
+
     public BrainParameters brainParameters = new BrainParameters();
     /**< \brief Defines brain specific parameters such as the state size*/
```

Comment:
in C# comments should come before the code being commented. This allows tools like Visual Studio or Ryder to automatically parse the doc and show it alongside intelisense results, etc.